modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/DiagramHeader.js

```diff
@@ -28,9 +28,6 @@ const DiagramHeader = ({
 	const [customColors, setCustoms] = useState(
```

Comment:
`const [customColors, setCustoms] = useState(` the setter must have the same name as the variable, so it must be `setCustomColors`